native/python_package/python_driver/requestprocessor.py

```diff
@@ -109,11 +118,24 @@ def _return_error(self, filepath: AnyStr='', status: AnyStr='error') -> None:
         :param status: error type, 'error' or 'fatal'
         """
 
+        if status == 'fatal':
+            ret_ast = None
+        elif ast:
+            ret_ast = ast
+        else:
+            # Empty modules are still modules
```

Comment:
I'm not sure if this function returns only errors or not. In the former case I'd say it's better to return an empty AST (or at least we should decide what to expect in that case and document it), in the latter not sure if this fallback case should be handled in this method (since its name suggests that it's for returning errors). Maybe the code is correct but the name should be changed.